<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Walter Roth</author>
 <comment>Preview Form for printing transaction data.</comment>
 <class>guiTransactionPrinterBase</class>
 <widget class="QWidget" name="guiTransactionPrinterBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>730</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GuiTransactionPrinterBase</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxOptions">
     <property name="title">
      <string>Print Options</string>
     </property>
     <property name="lineWidth" stdset="0">
      <number>1</number>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QGroupBox" name="groupBoxPages">
        <property name="toolTip">
         <string>Specify the pages to be printed. For reverse order, enter the higher number into the &quot;From Page&quot; box.</string>
        </property>
        <property name="title">
         <string>Pages to Print</string>
        </property>
        <layout class="QGridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="textLabel4">
           <property name="text">
            <string>Number of Pages:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="3">
          <widget class="QLabel" name="textLabelPages">
           <property name="text">
            <string>0</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QGroupBox" name="buttonGroup2">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QRadioButton" name="radioButtonAll">
              <property name="text">
               <string>Print &amp;All</string>
              </property>
              <property name="shortcut">
               <string>Alt+A</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonRange">
              <property name="text">
               <string>Print &amp;Range</string>
              </property>
              <property name="shortcut">
               <string>Alt+R</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel2">
           <property name="text">
            <string>From Page:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QSpinBox" name="spinBoxFrom">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>First page to be printed.</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="textLabel3">
           <property name="text">
            <string>To Page:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QSpinBox" name="spinBoxTo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Last page to be printed.</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxColor">
        <property name="toolTip">
         <string>Choose colors and font for printing.</string>
        </property>
        <property name="title">
         <string>Colors and Font</string>
        </property>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxLineColor">
           <property name="text">
            <string>Use different colors for even 
and uneven line numbers</string>
           </property>
           <property name="shortcut">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="buttonColor1">
           <property name="toolTip">
            <string>Selects the backgroundcolor for lines with uneven numbers.</string>
           </property>
           <property name="whatsThis">
            <string>Color selector buitton</string>
           </property>
           <property name="text">
            <string>Line Backgroundcolor &amp;1</string>
           </property>
           <property name="shortcut">
            <string>Alt+1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="buttonColor2">
           <property name="toolTip">
            <string>Selects the color for lines with even numbers.</string>
           </property>
           <property name="whatsThis">
            <string>Color selector button</string>
           </property>
           <property name="text">
            <string>Line Backgroundcolor &amp;2</string>
           </property>
           <property name="shortcut">
            <string>Alt+2</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="buttonFont">
           <property name="toolTip">
            <string>Select the font for printing.</string>
           </property>
           <property name="whatsThis">
            <string>Font selector button</string>
           </property>
           <property name="text">
            <string>&amp;Printer Font</string>
           </property>
           <property name="shortcut">
            <string>Alt+P</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxMargins">
        <property name="toolTip">
         <string>These are the margins on the paper.</string>
        </property>
        <property name="title">
         <string>Margins in cm</string>
        </property>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel2_2">
           <property name="text">
            <string>Left:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editLeft">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>32767</height>
            </size>
           </property>
           <property name="inputMask">
            <string>9.9; </string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="textLabel3_2">
           <property name="text">
            <string>Right:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="editRight">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>32767</height>
            </size>
           </property>
           <property name="inputMask">
            <string>9.9; </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel4_2">
           <property name="text">
            <string>Top:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="editTop">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>32767</height>
            </size>
           </property>
           <property name="inputMask">
            <string>9.9; </string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="textLabel5">
           <property name="text">
            <string>Bottom:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="editBottom">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>32767</height>
            </size>
           </property>
           <property name="inputMask">
            <string>9.9; </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QPushButton" name="buttonMarginsOk">
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Linewidth for transaction table</string>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QComboBox" name="comboBoxBorderWidth">
           <item>
            <property name="text">
             <string>No table borders</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Narrow table borders </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bold table borders</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>4</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>310</width>
       <height>310</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>32767</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel1">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>This is a preview, that shows what the printed pages will look like.</string>
        </property>
        <property name="text">
         <string>Print Preview</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="PreviewWidget" name="framePreview">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>122</red>
                <green>122</green>
                <blue>122</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>122</red>
                <green>122</green>
                <blue>122</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>122</red>
                <green>122</green>
                <blue>122</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>122</red>
                <green>122</green>
                <blue>122</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QPushButton" name="buttonHelp">
            <property name="text">
             <string>&amp;Help</string>
            </property>
            <property name="shortcut">
             <string>Alt+H</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonBack">
            <property name="toolTip">
             <string>Abort printing.</string>
            </property>
            <property name="text">
             <string>&amp;Cancel</string>
            </property>
            <property name="shortcut">
             <string>Alt+C</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>69</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonShowPrintOptions">
            <property name="toolTip">
             <string>Hide or show the comlete set of print options.</string>
            </property>
            <property name="text">
             <string>Show Print &amp;Options</string>
            </property>
            <property name="shortcut">
             <string>Alt+O</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>69</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPrinterSetup">
            <property name="toolTip">
             <string>Show the printer setup dialog. Select the printer here.</string>
            </property>
            <property name="text">
             <string>Printer &amp;Setup</string>
            </property>
            <property name="shortcut">
             <string>Alt+S</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPrint">
            <property name="toolTip">
             <string>Show printer setup dialog and print.</string>
            </property>
            <property name="text">
             <string>&amp;Print</string>
            </property>
            <property name="shortcut">
             <string>Alt+P</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>buttonShowPrintOptions</zorder>
       <zorder></zorder>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QFrame</extends>
   <header>previewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
